src/store/user/getters.js

```diff
@@ -3,6 +3,9 @@ import { Cookies } from 'quasar';
 const getters = {
   isLoggedIn(user) {
     return user.id !== null && Cookies.has('csrftoken');
+  },
+  getUserId(user) {
+    return user.id ? user.id.userId : undefined;
```

Comment:
I think you can do `user?.id?.userId || null`

Reply:
Since when is the safe-dereference-operator part of the language? It wasn't there in ES5, I guess. Is it safe to use on all browsers by now?

Reply:
I remember seeing it used somewhere, so I assumed it would work out of the box. Maybe it's a later ES version and requires Babel for proper browser support.